requirements.txt

```diff
@@ -1,4 +1,3 @@
 pyyaml
 msgpack
-joblib
```

Comment:
If we do want to support the optional use of joblib, I'm hesitant to use it as an opt-in dependency and would prefer it to be an opt-out dependency. This will reduce runtime significantly, particularly in scripted environments that only install the contents of **requirements.txt**.